### DISCUSSION TITLE
Why is my scroll-area not visible?

### DISCUSSION THREAD

**Discussion by AdagioB** [quasarframework/quasar#14394] [General - Components / Directives / etc] [answered]
I can get scroll area to work inside a q-drawer, but other than that, my scroll-area is too shy to show up. What am I doing wrong here?
It only works when I manually set the height of the scroll area, but I want it to use all available space

I tried a few things in this codepen

https://codepen.io/Adagio_B/pen/poVoEov

How do I accomplish this?

**Comment by pdanpdan** [comment, marked as answer]
Because of `min-height` - it will stretch to fill content (which would fill the parent ...)
https://codepen.io/pdanpdan/pen/LYmxpjx

**Reply by AdagioB** [reply]
Perfect, thanks a lot, you saved my day

**Reply by eugenevk** [reply]
The pen is no longer available. Can you please paste the code?
